TASVideos/wwwroot/js/site-head.js

```diff
@@ -64,6 +64,22 @@ window.addEventListener("DOMContentLoaded", function () {
             rate(btn.dataset.pubId, true);
         };
     });
+
+    // allow middle-clicking the search button to open in new tab
+    const searchBtnElem = document.querySelector(/*ul.navbar-nav */'form[action="/Search/Index"] button[type="submit"]');
+    // is there a function for this? reverse Element.querySelector?
+    let searchFormElem = searchBtnElem.parentElement;
```

Comment:
what happens if there is not a searchBtnElem on the page? this would be a javascript error.  Also this runs on every page load.  This needs to be fast, I'm not sure that query selector is.  Adding an id would be okay I think.

I also don't like this approach, or possibly any approach, the complexity vs value I think is questionable

Reply:
Is the search bar not on every page? I could easily handle that case if not.